ReactAndroid/src/main/java/com/facebook/react/views/art/ARTShapeShadowNode.java

```diff
@@ -182,6 +234,21 @@ protected boolean setupFillPaint(Paint paint, float opacity) {
               (int) (mFillColor[2] * 255),
               (int) (mFillColor[3] * 255));
           break;
+        case 1:
+          int stopsCount = getStopsCount(mFillColor);
+          int [] stopsColors = new int [stopsCount];
+          float [] stops = new float[stopsCount];
+          gradientStopsParsing(mFillColor, stopsCount, stops, stopsColors);
+          paint.setShader(
+            new LinearGradient(
+              mFillColor[3] * 2,
```

Comment:
Why do you multiply this by 2? Is this to compensate for the screen density? If so it shouldn't be handled this way as screen density is a device specific constant